test/location_test.cpp

```diff
@@ -58,21 +58,24 @@ TEST(LocationTest, RemoveAgent) {
 
 // all agents are removed
 TEST(LocationTest, ClearAgents) {
+    Location loc;
+    HoneyBee agent = HoneyBee();
+
     // empty location
-    location loc = Location();
+    loc = Location();
     EXPECT_NO_THROW(loc.clearAgents());
     EXPECT_EQ(loc.numAgents(), 0);
 
     // containing one agent
-    location loc = Location();
-    loc.addAgent(HoneyBee());
+    loc = Location();
+    loc.addAgent(agent);
     EXPECT_NO_THROW(loc.clearAgents());
     EXPECT_EQ(loc.numAgents(), 0);
 
     // containing multiple agents
-    location loc = Location();
-    loc.addAgent(HoneyBee());
-    loc.addAgent(HoneyBee());
+    loc = Location();
+    loc.addAgent(agent);
+    loc.addAgent(agent);
```

Comment:
Was supposed to be adding two different agents to a location, not the one agent twice